source/android/mobile/src/main/java/io/adaptivecards/adaptivecardssample/MainActivityAdaptiveCardsSample.java

```diff
@@ -703,7 +767,7 @@ public void onCardPayload(String cardPayload)
                         runOnUiThread(new Runnable() {
                             @Override
                             public void run() {
-                                loadAdaptiveCard(cPayload);
+                                loadAdaptiveCard("", cPayload);
```

Comment:
>loadAdaptiveCard [](start = 32, length = 16)

you could overload loadAdaptiveCard and loadHostConfig so this change is not needed